ci-operator/step-registry/telco5g/ptp/tests/run-in-container.sh

```
@@ -0,0 +1,276 @@
+else
+  USE_REMOTE=false
+  PODMAN="podman"
+fi
+
+# ---- Get telco-runner image: pull first, build as fallback ----
+REMOTE_IMAGE="registry.ci.openshift.org/ci/telco-runner:latest"
+if [[ "$BUILD_IMAGE" == "true" ]]; then
+  echo "Building telco-runner image (--build-image forced)..."
+  if [[ "$USE_REMOTE" == "true" ]]; then
+    REMOTE_BUILD_DIR="/tmp/ptp-ci-build-$$"
+    ssh "$REMOTE_HOST" "mkdir -p $REMOTE_BUILD_DIR"
```

Shellcheck (0.11.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)